coldfront/core/project/views.py

```diff
@@ -2403,32 +2403,42 @@ def show_pooled_project_selection_form_condition(wizard):
 class SavioProjectRequestListView(LoginRequiredMixin, TemplateView):
     template_name = 'project/project_request/savio/project_request_list.html'
     login_url = '/'
-
     # Show completed requests if True; else, show pending requests.
     completed = False
 
+    model = SavioProjectAllocationRequest
+
+    def get_queryset(self):
+        order_by = self.request.GET.get('order_by')
+        if order_by:
+            direction = self.request.GET.get('direction')
+            if direction == 'asc':
+                direction = ''
+            else:
+                direction = '-'
+            order_by = direction + order_by
+        else:
+            order_by = 'id'
+        return SavioProjectAllocationRequest.objects.order_by(order_by)            
```

Comment:
Something has been lost here--I noticed that the "Pending" and "Completed" views list the same requests.